Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -38,29 +38,27 @@ function Get-Certificates {
     [string[]] $CertificatesSASList = $CertificatesSASPlainString.Split(",", $options)
     Write-Host "Number of Certs passed $($CertificatesSASList.count)"
     if ($CertificatesSASList.count -eq 0) {
-        Write-Error "If adding an LDAPS identity source, please ensure you pass in at least one certificate" -ErrorAction Stop
```

Comment:
Aren't these the same thing when using the `erroraction -stop`

Reply:
Yes, but I think we should use `throw` because `ErrorAction` effect is frankly counter-intuitive. Was there an exception writing error? No, then why do you stop?!